packages/scss/src/components/form/component.scss

```diff
@@ -8,6 +8,10 @@
 			margin: 0;
 			padding: 0;
 			border: 0;
+
+			.form-field {
+				margin-block: var(--pr-t-spacings-50);
```

Comment:
Applying only a margin bottom on `.formLabel` and `.form-field` wouldn't be cleaner?

Reply:
That would work too, but it affects two elements instead of one.
The margins merge here, is that what's bothering you?

Reply:
I just wonder if margins collapse works well on every browsers